src/components/select/simple-select/simple-select.stories.mdx

```diff
@@ -0,0 +1,161 @@
+        <Option text='Black' value='2' />
+        <Option text='Blue' value='3' />
+        <Option text='Brown' value='4' />
+        <Option text='Green' value='5' />
+        <Option text='Orange' value='6' />
+        <Option text='Pink' value='7' />
+        <Option text='Purple' value='8' />
+        <Option text='Red' value='9' />
+        <Option text='White' value='10' />
+        <Option text='Yellow' value='11' />
+      </SimpleSelect>
+    </div>
+  </Story>
+</Preview>
+
+### Disabled:
+
+<Preview>
+  <Story name="disabled" parameters={{ info: { disable: true }}} >
+    <div style={{ height: '250px' }}>
+      <SimpleSelect name='disabled' id='disabled' defaultValue='3' disabled>
+        <Option text='Amber' value='1' />
+        <Option text='Black' value='2' />
+        <Option text='Blue' value='3' />
+        <Option text='Brown' value='4' />
+        <Option text='Green' value='5' />
+        <Option text='Orange' value='6' />
+        <Option text='Pink' value='7' />
+        <Option text='Purple' value='8' />
+        <Option text='Red' value='9' />
+        <Option text='White' value='10' />
+        <Option text='Yellow' value='11' />
+      </SimpleSelect>
+    </div>
+  </Story>
+</Preview>
+
+### Read Only:
+
+<Preview>
+  <Story name="readonly" parameters={{ info: { disable: true }}} >
+    <div style={{ height: '250px' }}>
+      <SimpleSelect name='readonly' id='readonly' defaultValue='4' readOnly>
+        <Option text='Amber' value='1' />
+        <Option text='Black' value='2' />
+        <Option text='Blue' value='3' />
+        <Option text='Brown' value='4' />
+        <Option text='Green' value='5' />
+        <Option text='Orange' value='6' />
+        <Option text='Pink' value='7' />
+        <Option text='Purple' value='8' />
+        <Option text='Red' value='9' />
+        <Option text='White' value='10' />
+        <Option text='Yellow' value='11' />
+      </SimpleSelect>
+    </div>
+  </Story>
+</Preview>
+
+### Transparent:
+
+<Preview>
+  <Story name="transparent" parameters={{ info: { disable: true }}} >
+    <div style={{ height: '250px' }}>
+      <SimpleSelect name='transparent' id='transparent' defaultValue='4' transparent>
+        <Option text='Amber' value='1' />
+        <Option text='Black' value='2' />
+        <Option text='Blue' value='3' />
+        <Option text='Brown' value='4' />
+        <Option text='Green' value='5' />
+        <Option text='Orange' value='6' />
+        <Option text='Pink' value='7' />
+        <Option text='Purple' value='8' />
+        <Option text='Red' value='9' />
+        <Option text='White' value='10' />
+        <Option text='Yellow' value='11' />
+      </SimpleSelect>
+    </div>
+  </Story>
+</Preview>
+
+### With type to search:
+
+<Preview>
+  <Story name="typetosearch" parameters={{ info: { disable: true }}} >
+    <div style={{ height: '250px' }}>
+      <SimpleSelect name='typetosearch' id='typetosearch' defaultValue='4' hasTypeToSearch>
```

Comment:
I think the `hasTypeToSearch` should be `filter`. We've not used the `has` prefix on props and I think people will confuse the word `search` when we come to implement the component that does a async search.

Reply:
actually we've already used `has` prefix, e.g.: 'hasSearchButton' in search or `hasAdditionalActions` in form. I could change it to `hasFilter` as it suggests it's a boolean.

Reply:
changed to `filterable` to keep backward compatibility

Reply:
I meant that `has` is only used within our internal components. `filterable` is fine.